examples/NIMF-Metal-2-AWS-connection/README.md

```diff
@@ -0,0 +1,13 @@
+# Metal to AWS Connection Example
```

Comment:
The name of this directory doesn't follow our example naming patterns. Should be all lowercase, kebab case (which it is) and the module should be the prefix. Rename directory to `metal-nimf-2-aws-connection/`